.github/scripts/install_slices/install_slices.py

```diff
@@ -328,11 +389,21 @@ def main() -> None:
                 )
             else:
                 install_slice(
-                    full_slice_name(pkg.package, slice),
+                    pkg.package,
+                    slice,
                     cli_args.arch,
                     cli_args.release,
+                    missing_copyright,
                 )
 
+            if any(missing_copyright.values()):
```

Comment:
Shouldn't the check here be `if !missing_copyright[full_slice_name(pkg.package, slice)]:`.

Reply:
this was actually too indented. We just need to check per pkg. 
This has been updated since then.